work_queue/src/work_queue.h

```diff
@@ -226,6 +226,10 @@ struct work_queue_stats {
 	timestamp_t time_internal;     /**< Total time the queue spents in internal processing. */
 	timestamp_t time_polling;      /**< Total time blocking waiting for worker communications (i.e., master idle waiting for a worker message). */
 	timestamp_t time_application;  /**< Total time spent outside work_queue_wait. */
+	timestamp_t prev_time_status_msgs; 
```

Comment:
As above. Remove these prevs. We do not want to modify work_queue_stats unless we really need to. Add a prev_stats to struct work_queue.